bootstrap/scripts/buildAll.js

```diff
@@ -13,9 +13,8 @@ const $ = (cmd) => {
 
 console.log("building all...");
 $(`npx lerna run build --scope @dendronhq/common-all`);
-$(
-  `npx lerna run build --parallel --scope "@dendronhq/{unified,common-server}"`
```

Comment:
Was this a merge conflict? Can we change it back to the --parallel version?

Reply:
I needed to build common-server first or it fails. I believe its b/c one of the unified files needs `DConfig` which I moved to common-server

Reply:
Synced up with Jon. Unified cannot import from common-server which means we cannot read configs in this package. Therefore I'm passing in the config to the proc methods along with the data (whereas before it used to be part of the engine data).

To ensure clients don't accidentally forget the config, I'm making config a required prop as well